build.gradle.kts

```diff
@@ -16,7 +16,7 @@ dependencies {
     testImplementation("org.projectlombok:lombok:latest.release")
 
     annotationProcessor("org.openrewrite:rewrite-templating:$rewriteVersion")
-    compileOnly("com.google.errorprone:error_prone_core:2.19.1:with-dependencies") {
+    compileOnly("com.google.errorprone:error_prone_core:2.19.1") {
```

Comment:
Otherwise the recipe will pick `error_prone_support` as the lib that provides the Guava classes, and put that in the generated template classpath leading to mismatches.